File: Benchmarks/enterprise/java/jakarta/testcode/BenchmarkTest22585.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import jakarta.ws.rs.*;
import jakarta.ws.rs.core.Context;
import jakarta.ws.rs.core.MediaType;
import jakarta.ws.rs.core.Response;

@Path("/")
public class BenchmarkTest22585 {

    private static String normalize(String v) { return v.strip(); }

    @GET
    @Path("/BenchmarkTest22585")
    @Produces(MediaType.APPLICATION_JSON)
    public Response BenchmarkTest22585(@HeaderParam("Origin") String origin, @Context HttpServletRequest request, @Context HttpServletResponse response) throws Exception {
        String originValue = origin != null ? origin : "";
        String data = normalize(originValue);
        request.setAttribute("hidden_form_field", data);
        request.getRequestDispatcher("/dashboard").forward(request, response);
        return Response.noContent().build();
    }
}

File: Benchmarks/enterprise/python/django/testcode/benchmark_test_22585.py
# SPDX-License-Identifier: Apache-2.0
from django.http import JsonResponse


def BenchmarkTest22585(request, path_param):
    path_value = path_param
    def normalize(value):
        return value.strip()
    data = normalize(path_value)
    return JsonResponse({'error': 'An internal error occurred'}, status=500)
